server/src/test/java/com/coincoinche/engine/CommonTestHelper.java

```diff
@@ -0,0 +1,100 @@
+package com.coincoinche.engine;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.coincoinche.engine.cards.Card;
+import com.coincoinche.engine.cards.Rank;
+import com.coincoinche.engine.cards.Suit;
+import com.coincoinche.engine.teams.Player;
+import com.coincoinche.engine.teams.Team;
+
+/**
+ * Several helper methods for initializing the game.
+ */
+public class CommonTestHelper {
```

Comment:
Clearer naming?
```suggestion
public class GameSetupTestHelper {
```

Reply:
It's not used only to set up the game though. I will change the javadoc comment which is outdated.